backend/entities/User.cs

```diff
@@ -14,6 +14,11 @@ public class User : BaseEntity
         /// </summary>
         public Guid Id { get; set; }
 
+        /// <summary>
+        /// get/set - Unique identifier to link user with Keycloak user.
+        /// </summary>
+        public Guid? KeycloakUserId { get; set; }
```

Comment:
Instead of using the `User.Id` it will now use the `User.KeycloakUserId` property to link the user to Keycloak.  This will allow copying data from other environments and make it easier to correct the link.